app.rb

```diff
@@ -24,31 +24,39 @@ def list_people
     puts ''
   end
 
-  def create_person # rubocop:disable Metrics/MethodLength
+  def create_person
     puts 'Which type of person you wish to create'
     puts '1. Student'
     puts '2. Teacher'
     print 'Enter selection: '
     person_type = gets.chomp.to_i
     print 'Name: '
-    name = gets.chomp
+    @name = gets.chomp
     print 'Age : '
-    age = gets.chomp.to_i
```

Comment:
- [ ] Each class should have only a single responsibility(ie Single responsibility rule). Therefore, the request for inputs should have been placed in the main.rb file or in another method/class whose single responsibility is to request inputs from the user. Kindly fix this.
- [**Optional**] I would kindly advise you to validate your inputs because right now, I can proceed with the creating of a person after entering an input that differs from 1 or 2. **Reminder**: These validations should be done in another method/class NOT in this method. We need to make sure that each method should carry out ONLY one aspect. [ReadMore...](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), Solid lesson